scripts/wheel/build_wheel_manylinux.sh

```diff
@@ -21,14 +21,25 @@ cp ../cmake/config.cmake .
 echo set\(RAF_USE_LLVM \"llvm-config-8\"\) >> config.cmake
 echo set\(HIDE_PRIVATE_SYMBOLS ON\) >> config.cmake
 echo set\(RAF_USE_GTEST ON\) >> config.cmake
-echo set\(RAF_USE_CUDA ON\) >> config.cmake
-echo set\(RAF_CUDA_ARCH 75\) >> config.cmake
-echo set\(RAF_USE_CUBLAS ON\) >> config.cmake
-echo set\(RAF_USE_CUDNN ON\) >> config.cmake
 echo set\(RAF_USE_SANITIZER OFF\) >> config.cmake
 echo set\(RAF_USE_MPI OFF\) >> config.cmake
 echo set\(RAF_USE_NCCL OFF\) >> config.cmake
-echo set\(RAF_USE_CUTLASS ON\) >> config.cmake
+
+# Set environment variables to specify the version and platform to build RAF
+# RAF_BUILD_VERSION. Option: [stable/nightly/dev]
+export RAF_BUILD_VERSION=dev
+
+# RAF_BUILD_PLATFORM. Option: [cpu/cu113]
+export RAF_BUILD_PLATFORM=cu113
```

Comment:
These are supposed to be set by users, so you only need to check whether these two environment variables are set already or not.